recipe/meta.yaml

```diff
@@ -10,15 +10,15 @@ package:
   version: {{ version }}
 
 source:
-  url: https://ambermd.org/downloads/AmberTools23_rc6.tar.bz2
-  sha256: debb52e6ef2e1b4eaa917a8b4d4934bd2388659c660501a81ea044903bf9ee9d
+  url: https://ambermd.org/downloads/AmberTools23.3.tar.bz2
```

Comment:
If there are full tarballs now, we need to remove the patch update logic from `build.sh`.

Reply:
We've never created these tarballs before.  The patch update logic will still work correctly and should stay there (it'll report previously-applied updates) because it will be needed in the future almost certainly.